classes/client_helper.php

```diff
@@ -88,6 +88,6 @@ public static function get_client(
      * @return client
      */
     public static function get_csv_client(): client_interface {
-        return new \enrol_oneroster\local\csv_client();
+        return new \enrol_oneroster\local\v1p2\csv_client();
```

Comment:
This needs to select v1p1 or v1p2 based on the selected get_config for integration version